cmd/notation/plugin/install.go

```diff
@@ -330,6 +331,16 @@ func installPluginWithOptions(ctx context.Context, opts plugin.CLIInstallOptions
 		if errors.As(err, &errPluginDowngrade) {
 			return fmt.Errorf("%w.\nIt is not recommended to install an older version. To force the installation, use the \"--force\" option", errPluginDowngrade)
 		}
+
+		var errExeFile *plugin.PluginExecutableFileError
+		if errors.As(err, &errExeFile) {
+			return fmt.Errorf("%w.\nPlease ensure the plugin is executable on %s/%s.", err, runtime.GOOS, runtime.GOARCH)
+		}
+
+		var errMalformedPlugin *plugin.PluginMalformedError
+		if errors.As(err, &errMalformedPlugin) {
+			return fmt.Errorf("%w.\nPlease ensure to install the plugin for %s/%s. Contact the plugin publisher if the plugin executable file is malformed.", errMalformedPlugin, runtime.GOOS, runtime.GOARCH)
```

Comment:
```suggestion
			return fmt.Errorf("%w.\nThe plugin is not compatible with platform %s and architecture %s. Contact the plugin publisher if the plugin executable file is malformed.", errMalformedPlugin, runtime.GOOS, runtime.GOARCH)
```

Reply:
@Two-Hearts Is this error message better?

Reply:
@yizha1 This is not necessarily true. The platform and arch can be correct, but the plugin executable file itself might be malformed.